erpnext/controllers/accounts_controller.py

```diff
@@ -1290,6 +1292,9 @@ def validate_quantity(child_item, d):
 				child_item.conversion_factor = 1
 			else:
 				child_item.conversion_factor = flt(d.get('conversion_factor'))
+		
+		if d.get("uom"):
+			child_item.uom = d.get("uom")
```

Comment:
- Fix: _Supplied Items_ table in Purchase order (subcontracting) does not get updated on changing UOM and conversion factor
- Add: Test for multi-uom in SO/PO (check if packed items is updated in SO and if supplied items is updated in PO)